### DISCUSSION TITLE
Move the screen in portrait mode?

### DISCUSSION THREAD

**Discussion by mtb0w** [ClassicOldSong/moonlight-android#13] [Q&A] [answered]
When I am streaming in portrait mode, it is in the middle of the screen. I'd like the screen to be at the top, with my on screen controls at the bottom, similar to a Gameboy SP layout 
![Screenshot_20240905_151245_Artemis](https://github.com/user-attachments/assets/9ba2add7-6406-461a-ae6e-7659c1db92b9)

**Comment by ClassicOldSong** [comment, marked as answer]
There's an option named `Top Center Display` in `Additional Settings` section.

**Reply by mtb0w** [reply]
Thank you! Do you have a suggested resolution that works well with GBA for this? 720p seems a little small

**Reply by ClassicOldSong** [reply]
Actually you can use the native resolution of GBA: 240x160
